mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -231,23 +304,98 @@ export class ExperimentListView extends Component {
 const classNames = {
   experimentListOuterContainer: {
     boxSizing: 'border-box',
-    marginLeft: '64px',
+    marginLeft: '32px',
+  },
+  experimentListItemContainer: {
+    display: 'inline',
+    marginLeft: '0px',
+    marginTop: '0px',
+    marginBotom: '0px',
+    marginRight: '5px',
+    paddingRight: '5px',
+  },
+  experimentLink: {
+    maxWidth: '100%',
+    width: '90px',
+    fontSize: '12px',
+    textAlign: 'left',
+  },
+  renameExperiment: {},
+  deleteExperiment: {
+    marginRight: '10px',
   },
   experimentListContainer: {
-    overflowY: 'scroll',
     overflowX: 'hidden',
-    width: ' 100%',
-    height: '90vh',
+    width: '100%',
     marginTop: '8px',
-    // Remove an empty space (transparent switcher) in the tree node to align the experiment name
-    // to the left.
-    '.du-bois-light-tree-switcher': {
+    '.ant-list-item-action': {
```

Comment:
Please create or find selectors that are not based on `ant`. The reason is that we're currently migrating a lot of UI pieces to internal design system - it also uses `ant` under the hood, but utilizes different prefixes basing on a used theme so we're moving away from `ant-` classes.